packages/eslint-plugin/tests/rules/explicit-function-return-type.test.ts

```diff
@@ -1008,24 +1110,27 @@ const func = (value: number) => ({ type: 'X', value } as Action);
       errors: [
         {
           messageId: 'missingReturnType',
-          line: 2,
-          endLine: 2,
+          line: 3,
+          endLine: 3,
           column: 14,
           endColumn: 32,
         },
         {
           messageId: 'missingReturnType',
-          line: 3,
-          endLine: 3,
+          line: 4,
+          endLine: 4,
           column: 14,
           endColumn: 32,
         },
       ],
     },
     {
-      filename: 'test.ts',
       code: `
 const func = (value: number) => ({ type: 'X', value } as const);
+      `,
+      // eslint-disable-next-line @typescript-eslint/internal/plugin-test-formatting
+      output: `
+const func = (value: number): { readonly type: "X"; readonly value: number; } => ({ type: 'X', value } as const);
```

Comment:
The `"X"` defaults to double quotes here, but the rest of the file uses single quotes. Is there a way for the fixer to know that?